process/transactionProcessor_test.go

```diff
@@ -122,7 +125,7 @@ func TestTransactionProcessor_SendTransactionComputeShardIdFailsShouldErr(t *tes
 		},
 		&mock.PubKeyConverterMock{},
 		hasher,
-		marshalizer,
+		marshalizer, funcNewTxCostHandler,
```

Comment:
add new param to new line (applicable in the entire file)

Reply:
done.